tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -431,7 +485,7 @@ static uint8_t get_oob_sc_local_data(const void *cmd, uint16_t cmd_len,
 }
 
 static uint8_t set_oob_sc_remote_data(const void *cmd, uint16_t cmd_len,
-				      void *rsp, uint16_t *rsp_len)
+					  void *rsp, uint16_t *rsp_len)
```

Comment:
And here. There seems to be so many places that I won't comment on all of them, but please fix them everywhere.